### PR TITLE
feat(LLM): multibuy 1.5 [LIVE-1710]

### PR DESCRIPTION
**Pull request by Justkant**
### ❓ Context

- **Impacted projects**: [e.g. `live-mobile`]
  
- **Linked resource(s)**: https://ledgerhq.atlassian.net/browse/LIVE-1710
Old PR: https://github.com/LedgerHQ/ledger-live-mobile/pull/2340

Implemented Multibuy-1.5 flow rebased on v3

### ✅ Checklist

- [ ] **Test coverage** 
- [ ] **Atomic delivery**: Depends on https://github.com/LedgerHQ/ledger-live/pull/91
- [x] **No breaking changes** 

### 📸 Demo

https://user-images.githubusercontent.com/3009753/169261798-55d273fd-4db2-428d-8cd0-b86ddad29ef8.mov

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

File: apps/ledger-live-mobile/android/settings.gradle [hunk 1]
```diff
@@ -6,6 +6,9 @@ project(':react-native-webview').projectDir = new File(rootProject.projectDir, '
 
 apply from: file("../node_modules/@react-native-community/cli-platform-android/native_modules.gradle"); applyNativeModulesSettingsGradle(settings)
 
+include ':flipper-plugin-rn-performance-android'
+project(':flipper-plugin-rn-performance-android').projectDir = new File(rootProject.projectDir, '../node_modules/flipper-plugin-rn-performance-android')
```
Comment (gre) [inline apps/ledger-live-mobile/android/settings.gradle:10]: this is unrelated to the PR / what is documented in description and should be shipped separately.

File: apps/ledger-live-mobile/src/icons/GooglePay.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+import Svg, { Path } from "react-native-svg";
+
+type Props = {
+  height: number,
+  width: number,
+};
+
+export default function GooglePay({ height = 12, width = 32 }: Props) {
+  return (
+    <Svg viewBox="0 0 32 12" width={width} height={height}>
```
Comment (gre) [inline apps/ledger-live-mobile/src/icons/GooglePay.js:12]: Svg can be complex for overall performance. We should aim to do one of these:
- keep using svg but memo that component
- use an asset instead. static image instead of component.

**Review by gre — COMMENTED** [review]
someone else have to continue the review as I didn't checked the business logic but just feedbacks from a high level perspective.

File: apps/ledger-live-mobile/android/settings.gradle (hunk 1 above)
Comment (Justkant) [reply apps/ledger-live-mobile/android/settings.gradle:10]: Yeah it was not my code at first, but after the rebase on LLM v3 I saw that too and wondered if I needed to clean that up.
I'll remove that

File: apps/ledger-live-mobile/src/icons/GooglePay.js (hunk 2 above)
Comment (Justkant) [reply apps/ledger-live-mobile/src/icons/GooglePay.js:12]: Btw, is there a reason we don't use https://react-svgr.com/ ?
With it we can generate components with memo from svg files and it supports RN

**Review by IAmMorrow — APPROVED** [review]

File: apps/ledger-live-mobile/src/components/FabActions.tsx [hunk 3]
```diff
@@ -140,7 +204,33 @@ const FabMarketActionsComponent = ({ currency, accounts, ...props }: Props) => {
                 NavigatorName.Exchange,
                 {
                   screen: ScreenName.ExchangeBuy,
-                  params: { currencyId: currency?.id, defaultAccount },
+                  params: {
+                    defaultTicker:
+                      currency &&
+                      currency.ticker &&
```
Comment (gre) [inline apps/ledger-live-mobile/src/components/FabActions.tsx:210]: `.ticker` can't be falsy 🤔

File: apps/ledger-live-mobile/src/screens/Exchange/ProviderList.js [hunk 4]
```diff
@@ -0,0 +1,256 @@
...
+export default function ProviderList({ route }: Props) {
+  const navigation = useNavigation();
+  const { t } = useTranslation();
+  const { colors } = useTheme();
+  const rampCatalog = useRampCatalog();
+  const { currency, type, accountId, accountAddress } = route.params;
+  const fiatCurrency = useSelector(counterValueCurrencySelector);
+
+  const filteredProviders = filterRampCatalogEntries(rampCatalog.value[type], {
+    cryptoCurrencies: currency.id ? [currency.id] : undefined,
```
Comment (gre) [inline apps/ledger-live-mobile/src/screens/Exchange/ProviderList.js:145]: both tokencurrency and cryptocurrency have an `.id` so this if can be simplified

File: apps/ledger-live-mobile/src/screens/Exchange/ProviderView.js [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+  return (
+    <>
+      <TrackScreen
+        category="Multibuy"
+        name="ProviderLiveApp"
+        provider={provider.appId}
+        trade={trade}
+      />
+      <WebPlatformPlayer
+        onClose={() => {}}
```
Comment (gre) [inline apps/ledger-live-mobile/src/screens/Exchange/ProviderView.js:65]: no need to pass a empty callback?

File: apps/ledger-live-mobile/src/screens/Exchange/SelectAccount.js [hunk 6]
```diff
@@ -24,40 +23,70 @@ import { formatSearchResults } from "../../helpers/formatAccountSearchResults";
...
+    mode = "buy",
+    currency,
+    analyticsPropertyFlow,
+    onAccountChange,
+  } = route.params;
 
   const accounts = useSelector(accountsSelector);
 
+  const availableAccounts = useMemo(
+    () => (currency ? getAccountTuplesForCurrency(currency, accounts) : []),
```
Comment (gre) [inline apps/ledger-live-mobile/src/screens/Exchange/SelectAccount.js:54]: according to the route, it's not possible for currency to be falsy

**Review by gre — APPROVED** [review]
setting it as approve, because not a blocker, but general quality to be improved in future
